### PR TITLE
Skip TTY checks

### PR DESCRIPTION
**Pull request by aymanbagabas**
Add `WithUnsafe` options to skip checking for TTY and running IoCtl commands.
This is useful for mocking console output and querying the terminal over SSH.

Fixes: https://github.com/muesli/termenv/issues/105 & https://github.com/muesli/termenv/pull/108

### CODE REVIEW

File: output.go [hunk 1]
```diff
@@ -99,6 +100,17 @@ func WithColorCache(v bool) func(*Output) {
 	}
 }
 
+// WithUnsafe returns a new Output with unsafe mode enabled. Unsafe mode doesn't
+// check whether or not the terminal is a TTY.
+//
+// This is useful when mocking console output and enforcing ANSI escape outpus
```
Comment (muesli) [inline output.go:106]: Typo in "outpus".

**Comment by heaths** [comment]
This works, but seems like sledgehammer to drive a thumbtack. Something like #108 is specific to the original issue in #105. This seems to be intended to just disable everything and isn't intuitive. How would one make the leap from "unsafe" to "tty" support?

**Comment by muesli** [comment]
I think we want both options, actually. I'm curious, what's your use-case for #108?

**Comment by heaths** [comment]
It's for tests. I've written a console module I use in various projects and much of it - or its direct dependencies - are based on termenv. The main purpose of that module is being able to test different output for CLIs, mainly for regression tests. I'd like to replace much of what I have rewritten myself with termenv (since it's already a common dependency), but being able to simulate TTY is critical to making that happen.

**Pull request merged**